src/test/java/com/synopsys/integration/blackduck/service/dataservice/UserServiceTestIT.java

```diff
@@ -30,123 +34,158 @@
 public class UserServiceTestIT {
     private static final IntHttpClientTestHelper INT_HTTP_CLIENT_TEST_HELPER = new IntHttpClientTestHelper();
 
+    private final BlackDuckServicesFactory BLACKDUCK_SERVICES_FACTORY = UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.createBlackDuckServicesFactory();
```

Comment:
the ALL_CAPS thing is usually reserved for static constants. I'd suggest making these static, or if that's not feasible, default to camelCase.